src/app/frontend/common/components/chips/component.ts

```diff
@@ -18,6 +18,8 @@ import {StringMap} from '@api/backendapi';
 
 import {ChipDialog} from './chipdialog/dialog';
 
+const truncateUrl = require('truncate-url');
```

Comment:
isn't it possible to use import here? 

In example
`import * as truncateUrl from 'truncate-url'`

Reply:
Unfortunately not. The typings are missing and the build fails.